pyanaconda/payload/dnf/payload.py

```diff
@@ -1540,8 +1554,14 @@ def update_base_repo(self, fallback=True, checkmount=True):
         # We need to check this again separately in case REPO_FILES were set above.
         if source_type in SOURCE_REPO_FILE_TYPES:
 
-            # If this is a kickstart install, just return now
-            if flags.automatedInstall:
+            # If this is a kickstart install, just return now as we normally do not
+            # want to read the on media repo files in such a case. On the other hand,
+            # the local repo files are a valid use case if the system is subscribed
+            # and the CDN is selected as the installation source.
+            subscription_proxy = SUBSCRIPTION.get_proxy()
```

Comment:
Same here.

Reply:
Changed here to move the check into the condition that checks if CDN is used as the installation source, so we can log an error message ig CDN source is used yet Subscription module is not available.